run_server.py

```
@@ -0,0 +1,63 @@
+        print("\n🏗️ Starting server...")
+        from src.api.main import app
+        
+        print("✅ Server ready!")
+        print("🌐 Server URL: http://localhost:8000")
+        print("📚 API Documentation: http://localhost:8000/docs")
+        print("🔍 Health Check: http://localhost:8000/health")
+        print("\n⚠️  Note: Using mock implementations for development/testing")
+        print("📝 All endpoints are functional with sample data")
+        print("\nPress Ctrl+C to stop the server")
+        
+        uvicorn.run(app, host="0.0.0.0", port=8000, reload=True, log_level="info")
```

Ruff (0.13.1)

Possible binding to all interfaces

(S104)